python/foglamp/common/plugin_discovery.py

```diff
@@ -41,11 +42,14 @@ def get_plugins_installed(cls, plugin_type=None, is_config=False):
             plugins_list.extend(plugins_list_c_north)
             plugins_list.extend(plugins_list_south)
             plugins_list.extend(plugins_list_c_south)
+            plugins_list.extend(plugins_list_filter)
             plugins_list.extend(plugins_list_c_filter)
             plugins_list.extend(plugins_list_c_notify)
             plugins_list.extend(plugins_list_c_rule)
         elif plugin_type in ['filter', 'notificationDelivery', 'notificationRule']:
             plugins_list = cls.fetch_c_plugins_installed(plugin_type, is_config)
+            plugins_list_filter = cls.fetch_plugins_installed("filter", is_config)
+            plugins_list.extend(plugins_list_filter)
```

Comment:
@pintomax please check 
https://github.com/foglamp/FogLAMP/pull/1732/files#diff-84ad01acf8d68fab247cedc5084a7a16L49-R51

if plugin_type in request is `filter` which now are expected to discover python based  plugins too `else` part will take care of it. 

Also fixed the unit tests.